### PR TITLE
Created cookbook for passing vectors/matrix to features object

### PR DESCRIPTION
**Pull request by yashdusing**
Feature/vector matrix cookbook
Issue #4105

### CODE REVIEW

File: doc/cookbook/source/examples/getting_started/passing_features_from_matrix.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Now we can pass the data from the vector to the features object.
+
+.. sgexample:: passing_features_from_matrix.sg:create_features
+
+----
+NOTE
+----
+
+It is important to note that conventionally, data is stored in a row-major fashion, meaning two row vector of size 3.
```
Comment (karlnapf) [inline doc/cookbook/source/examples/getting_started/passing_features_from_matrix.rst]: Suggestion: "Note that data is stored in [column major format](https://en.wikipedia.org/wiki/Row-_and_column-major_order), i.e. each column of the matrix corresponds to an observation / feature vector, where each vector consists of a number of variables that is equal to the number of rows of the matrix."
I.e. add the wikipedia link for explanation and for the feature vector

Comment (karlnapf) [reply doc/cookbook/source/examples/getting_started/passing_features_from_matrix.rst]: And remove the other sentences

File: doc/cookbook/source/examples/getting_started/passing_features_from_matrix.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+However, in Shogun, this will be column vectors of dimension 2. This is beacuse each data sample is stored in a column-major fashion,
+meaning each column here corresponds to an individual sample and each row in it to an atribute like BMI, Glucose concentration etc.
+Also, it is essential to map the data types of the features object & vector i.e one cannot pass a vector of integers to a method that expects floats
```
Comment (karlnapf) [inline doc/cookbook/source/examples/getting_started/passing_features_from_matrix.rst]: This is a good point. Maybe it should be phrased "Note that features are type-safe, e.g. you can only pass 64 bit floating point numbers to `RealFeatures`.

File: doc/cookbook/source/examples/getting_started/passing_features_from_matrix.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+----------
+References
+----------
+:wiki:`Feature (machine learning)`
```
Comment (karlnapf) [inline doc/cookbook/source/examples/getting_started/passing_features_from_matrix.rst]: good, also put the column major thing

File: examples/meta/src/getting_started/passing_features_from_vector.sg [hunk 4]
```diff
@@ -0,0 +1,16 @@
+Math:init_random(1)
+
```
Comment (karlnapf) [inline examples/meta/src/getting_started/passing_features_from_vector.sg]: I in fact suggest to remove the vector example

**Comment by yashdusing** [comment]
Thank you for the feedback ! I will try to solve this as soon as possible.
I'm still new to github & I couldn't find how to update existing PR . Will look into it.
Thanks once again !

**Comment by karlnapf** [comment]
http://lmgtfy.com/?q=update+existing+PR

**Pull request closed without merging**